src/types/mod.rs

```diff
@@ -540,17 +547,15 @@ pub enum ObjectUseInfo {
     None,
 }
 
-impl Default for ObjectUseInfo {
-    fn default() -> Self {
-        Self::None
-    }
-}
-
-impl ObjectUseInfo {
-    pub fn is_none(&self) -> bool {
-        matches!(self, Self::None)
-    }
-}
+/// Newtype around a collection of [`ObjectCandidate`] structs.
+///
+/// This abstracts away some common operations needed on this collection.  Because it needs
+/// to be passed through objects implementing the [`CacheItemRequest`] trait while avoiding
+/// copying and yet being able to mutate, we need to have inner mutability.
```

Comment:
this comment does not reflect reality, since this will be cloned, and it has no internal mutability

Reply:
Huh, yes.  the implementation changed a bit since this was written :/
thanks for the catch